mgr/src/server/datastore/ports.cr

```diff
@@ -39,4 +39,15 @@ module Datastore
     query = "DELETE FROM ports WHERE pool_id = ? AND node_id = ? AND updated_on < datetime('now', '-5 minutes')"
     connection.exec(query, pool_id, node_id)
   end
+
+  def delete_port(pool_id : String, distribute_groups : Array)
```

Comment:
change name to `delete_reserved_volume_ports`

Reply:
Done